raft/handler.go

```diff
@@ -12,11 +12,11 @@ import (
 // Handler represents an HTTP endpoint for Raft to communicate over.
 type Handler struct {
 	Log interface {
-		AddPeer(u *url.URL) (uint64, *Config, error)
+		AddPeer(u *url.URL) (uint64, uint64, *Config, error)
```

Comment:
It would be nice if these `uint64` params were named, so it was clearer what they were for, like the functions below.